compiler/rustc_middle/src/ty/context.rs

```diff
@@ -3,6 +3,7 @@
 use crate::arena::Arena;
 use crate::dep_graph::{self, DepGraph, DepKind, DepNode, DepNodeExt};
 use crate::hir::exports::ExportMap;
+use crate::hir::place;
```

Comment:
Sorry, I missed this earlier, can you do `use crate::hir::place::Place as HirPlace` similar to elsewhere. Place is commonly used to refer MIR Place.

Reply:
Done